alphadia/workflow/peptidecentric.py

```diff
@@ -587,6 +588,61 @@ def _skip_all_optimizers(
             )
             optimizer.skip()
 
+    def golden_section_search(self, optimizer):
```

Comment:
you should definitely explain the idea behin this somewhere in detail, maybe the docstring of this method would be a good place